SpatialGDK/Source/SpatialGDKServices/Private/Interop/Connection/EditorWorkerController.cpp

```diff
@@ -73,9 +73,8 @@ struct EditorWorkerController
 				"--existing_worker_id %s "
 				"--replacing_worker_id %s"), *ServicePort, *OldWorker, *NewWorker);
 		uint32 ProcessID = 0;
-		FProcHandle ProcHandle = FPlatformProcess::CreateProc(
-			*SpatialGDKServicesConstants::SpatialExe, *CmdArgs, false, true, true, &ProcessID, 2 /*PriorityModifier*/,
-			nullptr, nullptr, nullptr);
+		FProcHandle ProcHandle = SpatialCommandUtils::CreateSpatialProcess(CmdArgs, false, true, true, &ProcessID, 2 /*PriorityModifier*/,
```

Comment:
Does this need bIsInChina?

Reply:
Hmmm, perhaps. It was set to false in 0.8.1 so would it need to added here too? This is a local only command I believe. I'll add just in-case. If it is needed though then 0.8.1 needs work

Reply:
Actually the bIsInChina flag isn't available here. If this hasn't been flagged we can assume it's fine.